desktop-helper-sample/helper/public/server.js

```diff
@@ -11,8 +11,9 @@ const startServer = async () => {
 
   const io = new Server(server, {
     cors: {
-      origin: 'http://localhost:3000',
+      origin: '*',
       methods: ['GET'],
+      transports: ['websocket'],
```

Comment:
Hey @kerrishotts, just added some changes to this that (hopefully 🤞 ) enables the Socket.IO connection on Mac. Let me know if this change connects the two properly!

[Solution on CC Developer Forum](https://forums.creativeclouddeveloper.com/t/socket-io-client-doesnt-work/3355/9)

Reply:
It's alive! :-)